docs/source/user_guide/configuration_options.rst

```diff
@@ -942,7 +942,33 @@ Work Kubernetes
     work-kubernetes:
       - worktype: cat
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Kubernetes Environment Variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The following environment variables can be used to configure Kubernetes worker behavior:
+
+.. list-table:: Kubernetes Environment Variables
+    :header-rows: 1
+    :widths: auto
+
+    * - Variable
+      - Description
+      - Default value
+      - Type
+    * - ``RECEPTOR_KUBE_TIMEOUT_START``
+      - Initial timeout duration between Kubernetes API retry attempts (e.g., "1s", "500ms")
+      - 1s
+      - string (duration)
+    * - ``RECEPTOR_KUBE_RETRY_COUNT``
```

Comment:
This is still a fib back off right? 

If yes, then I think these docs need to be re-worded to explain the non linear effect adding a bigger number here will have